uamodeler/uamodeler.py

```diff
@@ -179,10 +179,10 @@ def _restore_state(self):
         self.resize(int(self.settings.value("main_window_width", 800)),
                     int(self.settings.value("main_window_height", 600)))
         #self.restoreState(self.settings.value("main_window_state", b"", type="QByteArray"))
-        self.restoreState(self.settings.value("main_window_state", b""))
-        self.ui.splitterLeft.restoreState(self.settings.value("splitter_left", b""))
-        self.ui.splitterRight.restoreState(self.settings.value("splitter_right", b""))
-        self.ui.splitterCenter.restoreState(self.settings.value("splitter_center", b""))
+        self.restoreState(self.settings.value("main_window_state", bytearray()))
```

Comment:
This stuff has been an issue in all pyqt apps I have seen. So far the only robust solution I found was to write something like:
val = self.settings.value("main_window_state", None)
if val is not None:
    self.restoreState(val)

Can you implement this? (or if you are sure your solution works on windows and with python2, we can try yours)
Thanks